app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt

```diff
@@ -2003,10 +2005,6 @@ open class EditorFragment : NavigationFragment<FragmentEditorBinding>(R.layout.f
         vm.onSelectProgrammingLanguageClicked(target, key)
     }
 
-    override fun onMoveToBinSuccess() {
```

Comment:
why has it been deleted?

Reply:
we don't need it anymore, navigation is used in that case